charts/ingress-nginx/templates/default-backend-deployment.yaml

```diff
@@ -5,6 +5,9 @@ metadata:
   labels:
     {{- include "ingress-nginx.labels" . | nindent 4 }}
     app.kubernetes.io/component: default-backend
+    {{- with .Values.defaultBackend.labels }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
```

Comment:
I'm curious if the above should also be included below at https://github.com/kubernetes/ingress-nginx/pull/6992/files#diff-b26819b1b2600e0b40ae78243b6b0efa9d34098f1594ac0838d23669bb93be18L23

Sure there are `podLabels`, but that would force the installer to duplicate some configuration if they should be the same.

Reply:
Done